docs/src/main/asciidoc/funqy-azure-functions-http.adoc

```diff
@@ -7,7 +7,7 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 :extension-status: preview
 include::_attributes.adoc[]
 :categories: cloud
-:summary: This guide explains Funqy's Azure Functions HTTP binding.
+:summary: Use Funqy HTTP binding with Microsoft Azure Functions to deploy your serverless Quarkus applications.  
```

Comment:
Looks like a very nice improvement overall but probably best to get rid of the trailing spaces here.

Reply:
Thanks, @gsmet.
I have now set "trim trailing whitespace" in my VS code so hopefully, no more whitespace churn from me in future PRs.